drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -23,6 +23,7 @@
 #include <hal/nrf_power.h>
```

Comment:
This shall be removed. No longer needed.

Reply:
done